BuildScript/Main/Remotion.Server.build

```diff
@@ -49,27 +49,44 @@
   
   <Target Name="TestBuild_Configuration">
     <PropertyGroup>
-      <Platforms>x86</Platforms>
-      <DatabaseSystems>NoDB+SqlServer2012</DatabaseSystems>
-      <Browsers>NoBrowser+Chrome</Browsers>
-      <ExecutionRuntimes>Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
-      <TargetRuntimes>NET48</TargetRuntimes>
-      <TestCategoriesToExclude></TestCategoriesToExclude>
+      <Configurations Condition="'$(Configurations)' == ''">Debug</Configurations>
+      <Platforms Condition="'$(Platforms)' == ''">x86</Platforms>
+      <DatabaseSystems Condition="'$(DatabaseSystems)' == ''">NoDB+SqlServer2012</DatabaseSystems>
+      <Browsers Condition="'$(Browsers)' == ''">NoBrowser+Chrome</Browsers>
+      <ExecutionRuntimes Condition="'$(ExecutionRuntimes)' == ''">Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
+      <TargetRuntimes Condition="'$(TargetRuntimes)' == ''">NET48</TargetRuntimes>
+      <TestCategoriesToExclude Condition="'$(TestCategoriesToExclude)' == ''"></TestCategoriesToExclude>
+      <SkipCleanProjects Condition="'$(SkipCleanProjects)' == ''">False</SkipCleanProjects>
+      <SkipTests Condition="'$(SkipTests)' == ''">False</SkipTests>
+      <SkipDocumentation Condition="'$(SkipDocumentation)' == ''">True</SkipDocumentation>
+      <SkipNuGetOrg Condition="'$(SkipNuGetOrg)' == ''">True</SkipNuGetOrg>
+      <SkipSourceLinks Condition="'$(SkipSourceLinks)' == ''">True</SkipSourceLinks>
+      <SkipNuGet Condition="'$(SkipNuGet)' == ''">True</SkipNuGet>
+      <SkipDependDB Condition="'$(SkipDependDB)' == ''">True</SkipDependDB>
     </PropertyGroup>
   </Target>
   
   <Target Name="FullBuild_Configuration">
     <PropertyGroup>
-      <Platforms>x86+x64</Platforms>
-      <DatabaseSystems>SqlServer2012+SqlServer2014+SqlServer2016+SqlServer2017</DatabaseSystems>
-      <Browsers>NoBrowser+Chrome+InternetExplorer+Firefox+Edge</Browsers>
-      <ExecutionRuntimes>Win_NET472+Win_NET48+EnforcedLocalMachine</ExecutionRuntimes>
-      <TargetRuntimes>NET472+NET48</TargetRuntimes>
-      <TestCategoriesToExclude></TestCategoriesToExclude>
+      <!-- By listing the debug-build first, the documentation will be generated based on the debug output. -->
+      <Configurations Condition="'$(Configurations)' == ''">Debug+Release</Configurations>
+      <Platforms Condition="'$(Platforms)' == ''">x86+x64</Platforms>
+      <DatabaseSystems Condition="'$(DatabaseSystems)' == ''">SqlServer2012+SqlServer2014+SqlServer2016+SqlServer2017</DatabaseSystems>
```

Comment:
```suggestion
      <DatabaseSystems Condition="'$(DatabaseSystems)' == ''">NoDB</DatabaseSystems>
```
The defaults for database and browser should get dropped since they reference IDs that are only set in the specific projects, so they don't make sense here any longer.